Documentation/Migration/Migration_Install.md

```diff
@@ -0,0 +1,52 @@
+# Migrate an ACPCore iOS app to the latest AEPCore
+
+Existing ACPCore users who are interested in upgrading to the latest AEPCore SDK, this guide describes how to upgrade. We have made upgrading seamless by providing a backward-compatible layer, which will ensure that an existing ACPCore implementation will be compatible with AEPCore.
+
+## Upgrading
+
+### Cocoapods
+
+The easiest way to upgrade from ACPCore to AEPCore is by using Cocoapods. To upgrade, update the reference to `ACPCore` in your projects `Podfile` as such:
+
+```diff
+- pod 'ACPCore'
++ pod 'ACPCore', :git => 'https://github.com/adobe/aep-sdk-compatibility-ios.git', :branch => 'main'
+```
+
+ Then run `pod install` and build your project, and you will be using the latest AEPCore SDK. Under the hood, this new podspec brings in the latest AEPCore SDK via Cocoapods.
+
+#### Version Control
+
+To control which version of AEPCore will be used when using this upgrade method, specify the versions in the `Podfile`.
+
+```ruby
+pod 'AEPServices', '1.0.0'
+pod 'AEPCore', '1.0.0'
+pod 'AEPLifecycle', '1.0.0'
+pod 'AEPIdentity', '1.0.0'
+pod 'AEPRulesEngine', '1.0.0'
+pod 'ACPCore', :git => 'https://github.com/adobe/aep-sdk-compatibility-ios.git', :branch => 'main'
+```
+
+### Manual Installation
+If Cocoapods is not an option, the compatibility layer can be built manually using the following commands:
+
+```bash
+git clone https://github.com/adobe/aep-sdk-compatibility-ios.git
+cd aep-sdk-compatibility
+make make-libs
+```
+After running these commands, the `.xcframework's` can be found within the build directory. Drag and drop these into an Xcode project and ensure that "copy items if needed" is checked.
```

Comment:
`.xcframework`s

Reply:
don't believe either is more correct AFAIK.

Reply:
well for sure there's no apostrophe 😛